ephios/static/plugins/basesignup/js/disposition.js

```diff
@@ -1,4 +1,22 @@
 $(document).ready(function () {
+    $("#id_user").djangoSelect2({
+        theme: "bootstrap5",
+        insertTag: function (data, tag) {
+            // Insert the tag at the end of the results
+            if (data.length === 0) {
+                data.push(tag);
+            }
+        },
```

Comment:
What does this do actually?

Reply:
the length check only shows the placeholder if there is no user matching the search string.
the data.push() adds the tag to the result list displayed to the user